app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -97,38 +97,48 @@ class FollowUsers extends Component {
     const { prev } = this.props;
     return (
       <div className="onboarding-main">
+        <Navigation prev={prev} next={this.handleComplete} />
         <div className="onboarding-content">
-          <h2>Ok, here are some people we picked for you</h2>
-          <div className="scroll">
-            <div className="select-all-button-wrapper">
-              <button type="button" onClick={() => this.handleSelectAll()}>
-                Select All 
-                {' '}
-                {selectedUsers.length === users.length ? '✅' : ''}
-              </button>
-            </div>
+          <header className="onboarding-content-header">
+            <h1 className="title">Suggested people to follow</h1>
+            <h3 className="subtitle">Let&apos; s review a few things first</h3>
```

Comment:
There's an extra space here in "Let's": `Let&apos; s` 😄

Reply:
Nice catch!